jest.config.js

```diff
@@ -1,3 +1,9 @@
 const jestConfig = require('@jahia/test-framework').jestConfig;
 
-module.exports = jestConfig;
+module.exports = {
+    ...jestConfig,
+    moduleNameMapper: {
+        ...jestConfig.moduleNameMapper,
+        '~/(.*)': '<rootDir>/src/javascript/$1'
```

Comment:
If I'm not mistaking you have the same exact line in content-editor, so instead of adding this here could you add it to the shared config and just update the test-framework version?

Reply:
You can add it to the test-framework version :)

Reply:
Since you are introducing it, you will have to do so :)

Reply:
Bye bye, you will fix this, I'm week off. Ciao :p